src/ray/raylet/placement_group_resource_manager_test.cc

```diff
@@ -154,7 +154,8 @@ TEST_F(NewPlacementGroupResourceManagerTest, TestNewMultipleBundlesCommitAndRetu
       std::make_shared<TaskResourceInstances>();
   ASSERT_TRUE(remaining_resource_scheduler->AllocateLocalTaskResources(
       init_unit_resource, resource_instances));
-  auto remaining_resouece_instance = remaining_resource_scheduler->GetLocalResources();
+  auto remaining_resouece_instance = remaining_resource_scheduler->GetLocalNodeResources();
```

Comment:
can we fix this typo in this PR? `remaining_resouece_instance` -> `remaining_resource_instance

Reply:
sure,  thanks for the remaining.